get-started/setup-an-ide.md

```diff
@@ -1,7 +1,90 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
 
+## GoLand
+
+You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB in many situations.
+
+> Goland uses [delve](https://github.com/go-delve/delve) to debug go programs, usually GoLand has an usable delve and needn't any additional configuration. But in some cases, you may need to download a delve by yourself.
+
+1. Make sure your `go` environment works well, see previous sections for more information.
+2. If you don't have GoLand, you just need to download it [here](https://www.jetbrains.com/go/download/) and install it.
+3. Open the TiDB project in GoLand.
```

Comment:
could you add some pictures here to help the audience be able to check whether their operations are correct?